Makefile

```diff
@@ -225,6 +225,20 @@ clean:
 protobuf:
 	$(MAKE) -C .. -f cockroach/build/protobuf.mk
 
+# The .go-version-check target is phony so that it is rebuilt every
+# time, but we create a .go-version-check file so that the include
+# directive below only complains about .go-version-check not existing
+# the first time we are run.
+.PHONY: .go-version-check
+.go-version-check:
+	@required=go1.6; \
```

Comment:
If we put the expected value "go1.6" in a separate file (`.go-version`? Is there any common convention we could follow for this?), then we could depend on that file in `sql/parser/Makefile` and remove the newly `.PHONY` targets.